### PR TITLE
Update sl-jdbc.md

### PR DESCRIPTION
**Pull request by rpourzand**
Draft PR

We now allow Dimension() to be used in a where clause for a time dimension (as opposed to TimeDimension ()). This matches the GROUP BY syntax.

We now have a way to use Order By when you are ordering by a dimension you operated on.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 1]
```diff
@@ -244,13 +244,13 @@ select * from {{
...
-Where filters have the following components that you can use:
+Where Filters have a few objects that you can use:
 
 - `Dimension()` - This is used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
 
-- `TimeDimension()` - This is used for all time dimensions and requires a granularity argument - `TimeDimension('metric_time', 'MONTH)`
+- `Entity()` - used for entities like primary and foreign keys - `Entity('order_id')`
 
-- `Entity()` - This is used for entities like primary and foreign keys - `Entity('order_id')`
+Note: If you prefer a more strongly typed `where` clause, you can optionally use the `TimeDimension` feature. This helps separate out categorical dimensions from time-related ones. The `TimeDimesion` input takes the time dimension name and also requires granularity, like this: `TimeDimension('metric_time', 'MONTH')`.
```
Comment (DevonFulcher) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: I don't think `strongly typed` is the best way to put it. `Dimension('metric_time').grain('week')` has similar typing.

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 2]
```diff
@@ -285,6 +285,20 @@ semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
   order_by=['order_gross_profit'])
   }}
 ``` 
+
+Note that for `order_by`, if you are operating on the object (e.g. changing granularity), you should use the following syntax:
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
When using `order_by` and making changes to the object (like changing granularity), use this syntax:
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md (hunk 1 above)
Comment (rpourzand) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Good call. Maybe "more explicit" is better? TimeDimension is more explicit than Dimension

Comment (DevonFulcher) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Ya, that is better!

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 3]
```diff
@@ -244,13 +244,13 @@ select * from {{
 
 Where filters in API allow for a filter list or string. We recommend using the filter list for production applications as this format will realize all benefits from the <Term id="predicate-pushdown"  /> where possible. 
 
-Where filters have the following components that you can use:
+Where Filters have a few objects that you can use:
 
 - `Dimension()` - This is used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
 
-- `TimeDimension()` - This is used for all time dimensions and requires a granularity argument - `TimeDimension('metric_time', 'MONTH)`
+- `Entity()` - used for entities like primary and foreign keys - `Entity('order_id')`
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
```

**Review by mirnawong1 — APPROVED** [review]
hey @rpourzand , lgtm!

**Comment by mirnawong1** [comment]
hey ya'll happy to merge this if this is good to go?

**Comment by mirnawong1** [comment]
Don't merge yet per Roxi

**Review by DevonFulcher — APPROVED** [review]

File: website/snippets/_sl-partner-links.md [hunk 4]
```diff
@@ -1,11 +1,23 @@
...
+1. **Hex**<br />
+To learn more about integrating with Hex, check out their [documentation](https://learn.hex.tech/docs/connect-to-data/data-connections/dbt-integration#dbt-semantic-layer-integration) for more info. Additionally, refer to [dbt Semantic Layer cells](https://learn.hex.tech/docs/logic-cell-types/transform-cells/dbt-metrics-cells) to set up SQL cells in Hex.
+
+1. **Google Sheets**<br />
+Google Sheets integration coming soon.
+
+1. **Tools that allows you to write SQL**<br />
+To connect to tools that allow you to write SQL, they must meet one of the two criteria: 
+   - Supports a generic JDBC driver option (such as DataGrip) or 
+   - Supports Dremio and uses ArrowFlightSQL driver version 12.0.0 or higher.
```
Comment (DevonFulcher) [inline website/snippets/_sl-partner-links.md]:
```suggestion
   - Uses Arrow Flight SQL JDBC driver version 12.0.0 or higher.
```
I would remove the Dremio part. While they also use AFS, we really have two separate products.

**Pull request merged**